src/sdk/python/examples/integrations/agent-network/charging/run.py

```diff
@@ -31,8 +31,8 @@
 logging.getLogger().setLevel(logging.INFO)
 
 class ChargingStationTalent(Talent):
-    def __init__(self, connection_string):
-        super(ChargingStationTalent, self).__init__('ChargingStationTalent', connection_string)
+    def __init__(self, protocol_gateway):
+        super(ChargingStationTalent, self).__init__('ChargingStationTalent', protocol_gateway)
```

Comment:
I think it should be `super().__init__` (to be aligned with the other python examples)